.eslintignore

```diff
@@ -3,3 +3,4 @@ examples
 templates
 __tests__/e2e/tmp/
 src/sdk/lh-trace-processor.ts
+test-projects/
```

Comment:
nit: I would suggest you to add these test-projects to global .gitignore and remove it from project specific settings as they are not related.